### PR TITLE
Cleanup unnecessary files

### PR DESCRIPTION
**Pull request by HLeithner**
Pull Request for Issue created in #31428 .

### Summary of Changes
Add unneeded files to the ignore list and remove them from git.

### Testing Instructions
Update with prebuilt package from 3.9.23-rc

### Actual result BEFORE applying this Pull Request
Files in /libraries/vendor/bin and other folders (see removed files in files tabs) exists

### Expected result AFTER applying this Pull Request
Got removed

### CODE REVIEW

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -2039,6 +2039,20 @@ public function deleteUnexistingFiles()
...
+			'/libraries/vendor/bin/lessify.bat',
+			'/libraries/vendor/bin/plessc',
+			'/libraries/vendor/bin/plessc.bat',
+			'/libraries/vendor/joomla/archive/.drone.jsonnet',
+			'/libraries/vendor/joomla/archive/.drone.yml',
+			'/libraries/vendor/joomla/string/.drone.jsonnet',
+			'/libraries/vendor/joomla/string/.drone.yml',
+			'/libraries/vendor/leafo/lessphp/.drone.yml',
+			'/libraries/vendor/leafo/lessphp/.phpunit.xml.dist',
+			'/libraries/vendor/leafo/lessphp/.ruleset.xml',
```
Comment (richard67) [inline administrator/components/com_admin/script.php]:
```suggestion
			'/libraries/vendor/leafo/lessphp/phpunit.xml.dist',
			'/libraries/vendor/leafo/lessphp/ruleset.xml',
```

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on b34c808bb92af88ca22981a3b533a32514159da0<br /><br />Tested with update from 3.9.23 RC to the update package built by Drone for this PR.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31440">issues.joomla.org/tracker/joomla-cms/31440</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on b34c808bb92af88ca22981a3b533a32514159da0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31440">issues.joomla.org/tracker/joomla-cms/31440</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31440">issues.joomla.org/tracker/joomla-cms/31440</a>.</sub>

**Comment by infograf768** [comment]
Tks

**Pull request merged**